silverback/_cli.py

```diff
@@ -87,10 +102,24 @@ def _network_callback(ctx, param, val):
     callback=cls_import_callback,
 )
 @click.option("-x", "--max-exceptions", type=int, default=3)
-@click.argument("path")
+@click.argument("path", required=False, type=str, default="bot")
 def run(cli_ctx, account, runner_class, recorder_class, max_exceptions, path):
     """Run Silverback application"""
 
+    def path_callback(path, calls=0):
```

Comment:
Let me know if this works for you, if you hate it, let me know how you'd like this to look